### DISCUSSION TITLE
WingPanel Customization

### DISCUSSION THREAD

**Discussion by arun0345** [pantheon-tweaks/pantheon-tweaks#128] [General] [answered]
Hi,
i'm using odin and have installed pantheon tweaks. but these tweaks having minimal features than elementary tweaks. for instance single click on files , wingpanel & dock tweaks. it would great to have included some kind of customization.

Thanks!!:)

**Comment by ryonakano** [comment, marked as answer]
Hi, thank you for using Pantheon Tweaks and opening a discussion!

We do our best to provide a way to change hidden settings in Pantheon Desktop Environment, but unfortunately some features that we rely on have been removed from Pantheon between the era of elementary OS Hera that elementary Tweaks works on and the era of Odin that Pantheon Tweaks works on. The most biggest example is the removal of the single click option in Files. We know this feature is really wanted from many users of Pantheon Tweaks and we are often asked where it'd gone in Pantheon Tweaks, but unfortunately this feature was removed from the source code of Files itself, which out of scope of our project. However, there is [a discussion](https://github.com/elementary/files/discussions/1729) open in the Files repository about problems introduced by this change, so this change *might* be reverted in the future release of Files, although I won't guarantee that it happens. But once this change is reverted in the Files side, we'll immediately revive the switch to toggle the single click option in Pantheon Tweaks.

Regarding to the settings for Wingpanel, I think the original elementary Tweaks also have only one settings that changes the maximum volume in the sound indicator. But we're open for your ideas of what kind of settings you want in Pantheon Tweaks. If so, feel free to ask us!

Also, regarding to the settings for the dock, we removed it because it'd been unused since the era of elementary OS Freya I think. See also for the discussion done when we performed this change: #24.

We're really proud that this just a small tool is loved from so many people. Thank you again for using Pantheon Tweaks! ❤️

**Reply by ChildishGiant** [reply]
Plank is getting a complete rewrite for 7 so there'd be no point adding support again just for it to all change

**Reply by arun0345** [reply]
Thank u for ur detailed explanation. i really appreciate u guys taking the time to create these tweaks.
 i just have few nice to have features.
1. Regarding wingpanel, it would be nice  if i can change the icon or panel size itself
2. In Tweaks Appearance, it would be good to have some extra themes / icons / cursors, other than what elementary offers

Thanks again for your hardwork!!

**Reply by ChildishGiant** [reply]
To add themes, icons and cursors [OpenDesktop](https://www.opendesktop.org/browse) is your best bet. You then place these downloads in /usr/share/themes or ~/.themes

**Reply by arun0345** [reply]
@ChildishGiant  i was talking about the out of the box support

**Reply by ryonakano** [reply]
Thanks for your idea. But I think we may not want to include some themes in Pantheon Tweaks by default, because themes which a person loves could different from the one thath another person loves; if we ship themes with Pantheon Tweaks, some user may like it but others may not and might say "I don't use it so it's redundant, remove it". So I think Pantheon Tweaks should persist in just a tool to change themes and let each user prepare themes they love.

However, reading your discussions I felt we may be able to prepare a link to instantly open the destination folder of themes; currently users need to open Files, make hidden files/folders appear, and then move the themes to the hidden destination folder. which may be a little bit annoying.
